src/components/CommunityCard/SocialLinks.astro

```diff
@@ -0,0 +1,45 @@
+---
+interface Props {
+  githubUrl?: string;
+  linkedinUrl?: string;
+  instagramUrl?: string;
+}
+
+const { githubUrl, linkedinUrl, instagramUrl } = Astro.props;
+---
+
```

Comment:
I'd like to share with you another way to import and use svg icons 🌱 :
1. Create a `public/icons` and add the attached svg icon (you can create svg files and paste into them the svg icons content we already have in `SocialMedia.astro` components, too)
![instagram](https://github.com/user-attachments/assets/d0791de2-3e9c-4491-8f8b-74f3be455355)
2. Import `instagram.svg` in `SocialLinks.astro` as described in this review comment
3. Use svg icon as described in the next review comment

```suggestion
   import instagram from '../../../public/icons/instagram.svg?raw'
```